src/pages/Signup.js

```diff
@@ -0,0 +1,170 @@
+import Avatar from '@mui/material/Avatar';
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import Link from '@mui/material/Link';
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import Typography from '@mui/material/Typography';
+import Image from 'mui-image';
+import dayjs, { Dayjs } from 'dayjs';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { margin } from '@mui/system';
+
+
+const theme = createTheme();
+
+function Signup() {
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        // TODO
+    };
+
+
+    const [dob, setDob] = React.useState(null);
+
+    return (
+        <ThemeProvider theme={theme}>
+            <Grid container component="main" sx={{ height: '100vh' }}>
+                <CssBaseline />
+                <Grid item xs={6} >
+                    <Box sx={{
+                        my: 8,
+                        mx: 4,
+                        p: 30,
+                        display: 'flex',
+                        flexDirection: 'column',
+                        alignItems: 'center',
+                    }}>
+                        <Typography component="h1" variant="h4">
+                            Sign up for Watmart!
+                        </Typography>
+                        <Image src="/logo/png/logo-no-background.png" fit="contain" duration={800} alt="" />
+                    </Box>
+                </Grid>
+                <Grid item xs={6} component={Paper} elevation={6} square>
+                    <Box
+                        sx={{
+                            my: 8,
+                            mx: 4,
+                            display: 'flex',
+                            flexDirection: 'column',
+                            alignItems: 'center',
+                        }}
+                    >
+                        <Avatar sx={{ m: 1, bgcolor: 'secondary.main' }}>
+                            <LockOutlinedIcon />
+                        </Avatar>
+                        <Typography component="h1" variant="h5">
+                            Sign up
+                        </Typography>
+                        <Box component="form" noValidate onSubmit={handleSubmit} sx={{ mt: 1 }}>
+                            <TextField
```

Comment:
Please wrap up your textfield / text input into a separate component, so that we can re-use it elsewhere in the app without re-implementing it